spark-atlas-connector/src/test/scala/com/hortonworks/spark/atlas/sql/InsertIntoHarvesterSuite.scala

```diff
@@ -0,0 +1,205 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hortonworks.spark.atlas.sql
+
+import java.io.File
+import java.util
+
+import scala.collection.JavaConverters._
+import scala.util.Random
+
+import org.apache.atlas.AtlasClient
+import org.apache.atlas.model.instance.AtlasEntity
+import org.apache.commons.io.FileUtils
+import org.apache.spark.sql.execution.datasources.InsertIntoHadoopFsRelationCommand
+import org.apache.spark.sql.execution.command.DataWritingCommandExec
+import org.apache.spark.sql.hive.execution.InsertIntoHiveTable
+import org.apache.spark.sql.SparkSession
+import org.scalatest.{BeforeAndAfterAll, Matchers, FunSuite}
+
+import com.hortonworks.spark.atlas.types.external
+
+class InsertIntoHarvesterSuite extends FunSuite with Matchers with BeforeAndAfterAll {
+
+  private var sparkSession: SparkSession = _
+  private val sourceHiveTblName = "source_" + Random.nextInt(100000)
+  private val sourceSparkTblName = "source_" + Random.nextInt(100000)
+  private val destinationHiveTblName = "destination_" + Random.nextInt(100000)
+  private val destinationSparkTblName = "destination_" + Random.nextInt(100000)
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    sparkSession = SparkSession.builder()
+      .master("local")
```

Comment:
Maybe, `local[1]`?

Reply:
Both local and local[1] will run Spark locally with one worker thread

Reply:
Oh, I see.